messenger.rst

```diff
@@ -643,6 +643,25 @@ You can limit the worker to only process messages from specific queue(s):
 To allow using the ``queues`` option, the receiver must implement the
 :class:`Symfony\\Component\\Messenger\\Transport\\Receiver\\QueueReceiverInterface`.
 
+.. _messenger-message-count:
+
+Knowing the number of messages in the "queue" for transport(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to know how many messages are in the "queues",
+you can do so with the command ``messenger:stats``:
+
+.. code-block:: terminal
+
+    # displays the number of messages in the "queue" for each all transports
+    $ php bin/console messenger:stats
+
+    # or specific transport(s) only
+    $ php bin/console messenger:stats my_transport_name other_transport_name
+
+This command won't work if the configured transport's receiver does not implement
```

Comment:
I would add it inside a note docblock, wdyt?

Reply:
Yes, why not? I also took the liberty of adding the note docblock at the end of the section 'Limit Consuming to Specific Queues' because I think we need one too. What do you think?